docs/rfcs/0001-expose-object.md

```diff
@@ -0,0 +1,43 @@
+# Summary
+
+Provide the build facility to expose objects of nebula for outside project.
+
+# Motivation
+
+Some outside project will use some components of nebula directly, such as cpp client will reference the `common` and `interface` code. And also useful to `UDF` and `extension` system etc. developing in future.
+
```

Comment:
Some outside projects will use some components of nebula directly, such as CPP client will reference the `common` and `interface` code. And also useful to `UDF` and `extension` systems etc. developing in the future.